rct229/rulesets/ashrae9012019/section19/section19rule9.py

```diff
@@ -58,25 +70,30 @@ def __init__(self):
                 },
             )
 
+        def is_applicable(self, context, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+            baseline_system_types_dict = data["baseline_system_types_dict"]
+
+            for key, value in baseline_system_types_dict.items():
+                if hvac_id_b in value:
+                    return True
+            return False
```

Comment:
The goal seems to be finding an HVAC system in the  baseline_system_types_dict

We typically use one return statement in the end instead of multiple return statements unless we are in a complex scenario (nested if for example).

```python
return any(hvac_id_b in baseline_system_types_dict[system_type] for system_type in baseline_system_types_dict.keys())
```
This shall do the trick.

Reply:
Good point! Addressed.